Sources/SwiftPlot/HistogramSeries.swift

```diff
@@ -1,5 +1,7 @@
-public struct HistogramSeries<T> where T: Comparable {
-    public var data = [T]()
+public class HistogramSeries<T> where T: Comparable {
```

Comment:
Why has this become a class?

Reply:
Oh I’ll revert it back to a struct, I completely missed that after merging from master, I will also take a look at the rest to see if anything else got changed that shouldn’t have.